src/consensus/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2023-present The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or https://opensource.org/license/mit/.
+
+# Must be included before CMAKE_INSTALL_INCLUDEDIR is used.
+include(GNUInstallDirs)
+
+add_library(bitcoinconsensus
+  ../script/bitcoinconsensus.cpp
+  ../support/cleanse.cpp
+)
+target_compile_definitions(bitcoinconsensus
+  PRIVATE
+    BUILD_BITCOIN_INTERNAL
+)
+target_include_directories(bitcoinconsensus
+  PUBLIC
+    $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/src>
+    $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
+)
+target_link_libraries(bitcoinconsensus
+  PRIVATE
+    core
+    lib
+    bitcoin_consensus_sources
+    bitcoin_crypto_sources
+    secp256k1
+)
+set_target_properties(bitcoinconsensus PROPERTIES
+  SOVERSION 0
+  VERSION 0.0.0
+)
+
+install(TARGETS bitcoinconsensus
+  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+  ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+)
+install(FILES bitcoinconsensus.h
```

Comment:
Should be `../script/bitcoinconsensus.h`?
Might also be good to add an install step to the CI.

Reply:
> Should be `../script/bitcoinconsensus.h`?

Fixed.

> Might also be good to add an install step to the CI.

Added.